test/onhost-e2e/ansible/ebpf_agent.yaml

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
I wanted to check the execution but I've not found it, was it tested in a previous commit?